packages/devextreme/js/ui/popup/ui.popup.js

```diff
@@ -510,7 +518,9 @@ const Popup = Overlay.inherit({
             toolbarsItems.push(this._getCloseButton());
         }
 
-        return toolbarsItems;
+        const sortedActionItems = sortActionButtonsItems(doneCancelButtonsInfo).map(item => item.item);
```

Comment:
Naming is misleading for me. It seems like we name similar instances different way: "actionItems", "toolbarItems", "actionButtons", "doneCancelButtons", "applyValueButtons". Can we unify the approaches?